test/filesystems/blk_filesystem_test.rb

```diff
@@ -88,4 +88,28 @@
       end
     end
   end
+
+  describe "#in_network?" do
```

Comment:
Can I suggest to move this test to `test/filesystems/base` and then have scenarios like these altogether?
```
for a NFS filesystem
  it returns true
for a filesystem in an LVM logical volume
  if any of the PVs is a network device
    it returns true
  if all the PVs are local
    it returns false
for a filesystem in a partition
  if the partition belongs to a local disk
     it returns false
  if the partition belongs to a DASD
     it returns false
  if the partition belongs to a network disk
     it returns true
for a filesystem directly in a network disk
  it returns true
for a filesystem directly in a local disk
  it returns false
```
Maybe it's a little bit too much, but the idea is that anybody reading the output knows exactly what to expect from the method (that's where the "spec" of RSpec comes from :wink:) and with your output I wouldn't have a clear picture of the method being also available for NFS or about what happens with LVM.